packages/shared/world/parcelSceneManager.ts

```diff
@@ -83,7 +82,7 @@ export async function enableParcelSceneLoading(options: EnableParcelSceneLoading
   ret.on('Scene.shouldStart', async (opts: { sceneId: string }) => {
     const sceneId = opts.sceneId
     const parcelSceneToStart = await ret.getParcelData(sceneId)
-
+    ;(global as any)['globalStore'].dispatch({ type: 'Loading scene', sceneId })
```

Comment:
Use helper functions here?
```suggestion
    globalSignalSceneLoad(sceneId)
```